policybot/dashboard/topics/postsubmit/topic.go

```diff
@@ -173,7 +201,7 @@ func (ps *PostSubmit) getLabelEnvTable(context context.Context, baseSha string)
 	return summary, nil
 }
 
-func (ps *PostSubmit) getLabelTree(input map[string]map[string]int, envNames map[string]int) LabelEnvSummary {
+func (ps *PostSubmit) getLabelTree(input map[string]map[string]int, envNames map[string]int, depth int) LabelEnvSummary {
```

Comment:
Do we really need to add the depth variables here? Couldn't we just pass them in as parameters into the nested template? https://www.calhoun.io/intro-to-templates-p2-actions/, section "Template Variables"